coordinator/core/core.go

```diff
@@ -75,6 +75,13 @@ const coordinatorName string = "Marblerun Coordinator"
 // coordinatorIntermediateName is the name of the Coordinator. It is used as CN of the intermediate certificate which is set when setting or updating a certificate.
 const coordinatorIntermediateName string = "Marblerun Coordinator - Intermediate CA"
 
+// storage keys for the certificates used in the Coordinator
+const (
+	rootCAStore         string = "rootCA"
+	intermediateCAStore string = "intermediateCA"
+	marbleRootCAStore   string = "marbleRootCA"
```

Comment:
```suggestion
	storeRootCA         = "rootCA"
	storeIntermediateCA = "intermediateCA"
	storeMarbleRootCA   = "marbleRootCA"
```
maybe prefix this to better group them, considering that this will grow?